ethcore/light/src/types/request/mod.rs

```diff
@@ -350,6 +364,33 @@ impl IncompleteRequest for Request {
 			Request::Execution(req) => req.complete().map(CompleteRequest::Execution),
 		}
 	}
+
+	fn adjust_refs<F>(&mut self, mapping: F) where F: FnMut(usize) -> usize {
+		match *self {
```

Comment:
Yes, although there are currently tests for behavior of the individual types as well as their enum-wrapped counterparts